### DISCUSSION TITLE
Setup only succeeds when i use airgap ......

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#5134] [General] [answered]
securityonion-2.3.61-STENODOCKER.iso  
Hi, I have read that airgap is not as secure as some think it is. But it is the only way i can get my SO to setup. 
 
  thanks for any advice

**Comment by iqworks** [comment]
WHOA, it completed successfully with a custom type install !!!!
what is the best way to install and setup:
typical.
custom.
airgap. 
  i may be wrong but i think that using airgap is not as good as installing without it, but, i dont know which is best. which is best practice to instal with (typical, custom or airgap). 
Why did custom work and typical not work?
i am new to SO and VMs, i could really use some clearity on these things. 
  it seems to me that many of the videos i followed to install SO used typical. 
I feel like i have taken over this forum :-) , sorry, i just want to learn how to call up that SOC. 
  thanks for any advice or suggestions

**Comment by dougburks** [comment, marked as answer]
> what is the best way to install and setup

There is no single best way for all users. If there was a single best way, the installer wouldn't ask you the question and it would automatically configure it for you. Those options are there because different users on different networks need those different options. It's the user's responsibility to understand their network and choose the correct option. The airgap option is intended for users on airgap networks that do not have Internet access.  If you DO have Internet access, then you don't need to choose the Airgap option, but you DO need to make sure that your Security Onion installation has a proper IP address and can route traffic to the Internet.

> I feel like i have taken over this forum

Yes, it does feel like that.  You've created multiple posts and most of them are asking basic networking questions that are beyond the scope of this discussion forum. Please be respectful of others and keep the signal-to-noise ratio as high as possible as described in the [discussion forum guidelines](https://github.com/Security-Onion-Solutions/securityonion/discussions/1720).

Thanks!

**Reply by iqworks** [reply]
Hi Doug, thanks for taking the time to give me info in airgap. I see now that airgap is really meant to be used if you dont have access to the internet. I am working on how to use the right IP to route that traffic.

 I am only trying to find out the most common way the majority of people use SO. 
  I thought my questions were directly or indirectly related to security onion? If you could give me an example of something i asked that i should not have? I need this forum so i will try to watch my questions. 
  I was directed to those SO forum rules before. Not sure how you mean "signal-to-noise ratio as high". I googled it and it said something about decibels. I am thinking it means something about the frequency of questions and/or the pertinence of questions? 
 As i mentioned i DO apoligize for my rookie questions and **_NEVER_** intend to be disrespectful to anyone. Different forums allow different questions, i didnt know there were questions this forum requires you not to ask when they are ultimately related to SO. However, when your new to SO, it has a learning curve and we all have to start somewhere when it comes to putting things together. Not sure what the networking, cyber security or SO competency level I should be at for this forum? But I am new to all three of these.

  SO is not as straight forward as some other VMs, but i think the variety of environments it offers is why its so popular and thats why I will learn it. 
  There is so much SO documentation that it takes time to find out basic info about how some things relate. And linux grows so fast that some youtube videos are obsolete and recent ones show how to setup SO in different ways, so please excuse my confusion.
  But I am getting the hang of it slowly. 
  I am anxious for the time when i will be more proficient at SO, then I might be able to help others learn as well. 

with respect, thanks for your advice!

**Reply by dougburks** [reply]
> Not sure what the networking, cyber security or SO competency level I should be at for this forum? But I am new to all three of these.

Most folks learn operating systems, virtualization, and networking before trying to learn cyber security. There is a good reason for this. When Security Onion generates an alert or log, you must understand all of these topics before you can properly interpret the data and understand how to respond to it.

**Reply by mishrapcdac** [reply]
my system gets hanged  at 66% when i launch standalone for production use in security onion why?? please help me .

**Reply by mishrapcdac** [reply]
how to setup security onion modules? with internet or without internet ??

i am using internet using static ip 

but when that installation goes to 66% my system is hanged why please help me?

**Comment by iqworks** [comment]
Yea, your right. I don’t want to bother you with this anymore. You are simply saying that networking is a subject and SO is a tool and this forum is for SO. 

Even though I have been a software developer since  1982 (I am 66), I am a certified java programmer and a Microsoft certified professional developer (and a certified personal trainer – should of gotten a degree in something). That virtually nothing to do with networking. 

  This past June I received my cybersecurity certificate from UCLA. It was a months of a combination of classes. This networking stuff was a surface study and mixed in with building a home lab. Not enough time to be good at networking and I only did what I could to pass assignments and tests 😊. This is August, I am JUST beginning to REALLY learn how to use these concepts and tools in the field so to speak. I was told that it could take a while to get good at it. 

   One of the things I have learned, thanks to you, is SO Forum etiquette.  

   I really hope to be one of your forum members that can contribute one day. 

   So, I really appreciate the help I get from the SO community. And I hope to be good at this soon. Wish me luck. 

    Thank you so much Doug.

**Reply by dougburks** [reply]
Thanks for understanding and good luck!

**Reply by mishrapcdac** [reply]
how to setup security onion modules? with internet or without internet ??

i am using internet using static ip 

but when that installation goes to 66% my system is hanged why please help me?

**Reply by dougburks** [reply]
@mishrapcdac Please do not reply to somebody else's discussion with your own questions. Please review the guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720. Thanks!

**Comment by iqworks** [comment]
Hi Doug, I got the email from mishrapcdac, but I don’t remember replying? 

thanks

**Reply by dougburks** [reply]
@iqworks As it says below in the Github email itself:
```
You are receiving this because you authored the thread.
```

Again, please keep in mind that generic questions about email or Github are beyond the scope of this Security Onion discussion forum.

This discussion is now locked.
